checker/src/org/checkerframework/checker/tainting/qual/Tainted.java

```diff
@@ -13,7 +13,7 @@
  * This annotation is associated with the {@link  org.checkerframework.checker.tainting.TaintingChecker}.
  *
  * @see Untainted
- * @see  TaintingChecker
+ * @see org.checkerframework.checker.tainting.TaintingChecker
```

Comment:
Yes, qualifiers are compiled separately from the rest of the framework to ensure that checker-qual.jar is self contained.  So, if TaintingChecker is imported rather than fully-qualified, compiling checker-qual fails.